linter/lintfuncs.go

```diff
@@ -12,11 +12,31 @@ const (
 	ThisUpdateLimitSubscriber = "96h" // 4 days
 	ProducedAtLimitCA = "8760h" // 365 days
 	ThisUpdateLimitCA = "8760h" // 365 days
+	NextUpdateLimitSubscriber = "240h" // 10 days
 )
 
+// DurationToString is a map mapping durations to more readable strings
 var DurationToString = map[string]string {
 	ProducedAtLimitSubscriber: "4 days",
 	ProducedAtLimitCA: "365 days",
+	NextUpdateLimitSubscriber: "10 days",
+}
+
+// CheckSignature checks in the ocsp response is signed with an algorithm that uses SHA1
+// Source: Apple Lints 10 & 12
+func CheckSignature(resp *ocsp.Response, leafCert *x509.Certificate) (LintStatus, string) {
+	if resp.Signature == nil || len(resp.Signature) == 0 {
+		return Failed, "OCSP Response is not signed"
+	}
+
+	algo := resp.SignatureAlgorithm
+
+	// These are all the SHA1 based algorithms, see https://godoc.org/crypto/x509#SignatureAlgorithm
+	if algo == x509.SHA1WithRSA || algo == x509.DSAWithSHA1 || algo == x509.ECDSAWithSHA1 {
+		return Failed, "OCSP Response is signed with an algorithm that uses SHA1"
+	}
+	
+	return Passed, "OCSP Response is signed with an algorithm that does not use SHA1"
```

Comment:
Where is `Passed` defined? Is this in another module. I'm not sure what go conventions are, but maybe since this is a constant it should be in all caps to let us know it isn't defined in the function. Like `PASSED`. The same with `Failed -> FAILED`.

Reply:
Passed is defined in `linter.go`, and since they are all of the same `linting` package, they share all exported definitions with one another.

Constants in Golang are typically stylized with camelcase: https://stackoverflow.com/questions/22688906/go-naming-conventions-for-const